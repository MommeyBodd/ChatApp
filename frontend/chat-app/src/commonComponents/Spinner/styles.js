import { makeStyles } from "@material-ui/core/styles";

export const useStyles = makeStyles(theme => ({
  root: {
    display: "flex",
    "& > * + *": {
      marginLeft: theme.spacing(2)
    },
    position: "absolute",
    top: "50%",
    left: "50%"
  }
}));
