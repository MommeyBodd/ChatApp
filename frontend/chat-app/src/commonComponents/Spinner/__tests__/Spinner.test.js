import React from "react";
import { Spinner } from "../Spinner";

describe("Renders", () => {
  test("should render correctly", () => {
    const wrapper = shallow(<Spinner />);

    expect(wrapper).toMatchSnapshot();
  });
});
