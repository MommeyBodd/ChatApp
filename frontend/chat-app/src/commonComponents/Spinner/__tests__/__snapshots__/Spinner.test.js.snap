// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders should render correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(CircularProgress)) />
</div>
`;
